src/Magnum/DartIntegration/Test/WorldGLTest.cpp

```diff
@@ -0,0 +1,514 @@
+    shapeNode->getVisualAspect()->setColor(dart::Color::Blue());
+    shapeNode->setRelativeTransform(tf);
+
+    /* Set the geometry of the Body */
+    setGeometry(bn);
+    return bn;
+}
+
+/* Add a soft body with the specified Joint type to a chain */
+template<class JointType>
+dart::dynamics::BodyNode* addSoftBody(const dart::dynamics::SkeletonPtr& chain, const std::string& name, dart::dynamics::BodyNode* parent = nullptr)
+{
+    constexpr double default_shape_density = 1000; // kg/m^3
+    constexpr double default_shape_height  = 0.1;  // m
+    constexpr double default_shape_width   = 0.03; // m
+    constexpr double default_skin_thickness = 1e-3; // m
+
+    constexpr double default_vertex_stiffness = 1000.0;
+    constexpr double default_edge_stiffness = 1.0;
+    constexpr double default_soft_damping = 5.0;
+
+    /* Set the Joint properties */
+    typename JointType::Properties joint_properties;
+    joint_properties.mName = name+"_joint";
+    if(parent)
+    {
+        /* If the body has a parent, we should position the joint to be in the
+         * middle of the centers of the two bodies */
+        Eigen::Isometry3d tf(Eigen::Isometry3d::Identity());
+        tf.translation() = Eigen::Vector3d(0, 0, default_shape_height / 2.0);
+        joint_properties.mT_ParentBodyToJoint = tf;
+        joint_properties.mT_ChildBodyToJoint = tf.inverse();
+    }
+
+    /* Set the properties of the soft body */
+    dart::dynamics::SoftBodyNode::UniqueProperties soft_properties;
+    /* Make a wide and short box */
+    double width = default_shape_height, height = 2*default_shape_width;
+    Eigen::Vector3d dims(width, width, height);
+
+    double mass = 2*dims[0]*dims[1] + 2*dims[0]*dims[2] + 2*dims[1]*dims[2];
+    mass *= default_shape_density * default_skin_thickness;
+    soft_properties = dart::dynamics::SoftBodyNodeHelper::makeBoxProperties(dims, Eigen::Isometry3d::Identity(), Eigen::Vector3i(4,4,4), mass);
+
+    soft_properties.mKv = default_vertex_stiffness;
+    soft_properties.mKe = default_edge_stiffness;
+    soft_properties.mDampCoeff = default_soft_damping;
+
+    /* Create the Joint and Body pair */
+    dart::dynamics::SoftBodyNode::Properties body_properties(dart::dynamics::BodyNode::AspectProperties(name), soft_properties);
+    dart::dynamics::SoftBodyNode* bn = chain->createJointAndBodyNodePair<JointType, dart::dynamics::SoftBodyNode>(parent, joint_properties, body_properties).second;
+
+    /* Zero out the inertia for the underlying BodyNode */
+    dart::dynamics::Inertia inertia;
+    inertia.setMoment(1e-8*Eigen::Matrix3d::Identity());
+    inertia.setMass(1e-8);
+    bn->setInertia(inertia);
+
+    return bn;
+}
+
+}
+
+struct DartIntegrationTest: Magnum::OpenGLTester {
+    explicit DartIntegrationTest();
+
+    void test();
+    void test_soft();
+    void urdf();
+    void test_multi_mesh();
+    void texture();
+
+    void bench_simple();
+    void bench_soft();
```

Comment:
Coding style: use `camelCase` instead of `underscore_case`, please. Also no need to prefix with `test` or `bench`, it's clear enough from the test output :)